tests/fixtures/docker.py

```diff
@@ -72,7 +72,6 @@ def cleanup_all_new_docker_objects(docker: DockerClient, worker_id: str):
             logger.warning("Failed to clean up Docker objects")
 
 
-@pytest.mark.timeout(120)
```

Comment:
This was the only specific pytest failure I could find. The documentation says that marks never had any effect on fixtures (https://docs.pytest.org/en/latest/deprecations.html#applying-a-mark-to-a-fixture-function). Running the full suite locally I saw tests using this timing out, but unsure if that's because of this missing mark (which shouldn't have been doing anything) or just general slowness.